pkg/storage/db_test.go

```diff
@@ -1429,8 +1453,14 @@ func TestForceSchemaToVersionSuccess(t *testing.T) {
 	version := uint(rand.Uint64())
 	migrator := NewMockStorageMigrator(ctrl)
 	migrator.EXPECT().Force(int(version)).Return(nil)
-	db := &DB{migrator: migrator}
-	err := db.ForceSchemaToVersion(context.Background(), version)
+	mockdb, mock, err := sqlmock.New()
```

Comment:
nit: this feels quite repetitive(not DRY) although I noticed the rest o0f the tests in this file have significant duplication as well. A future improvement would be creating some kind of shared testing setup and fixtures.

Reply:
We historically have lower expectations for code quality in tests (AFAIU the linting does not cover them much either), so it is a good thing to think about improvements. TBH I was thinking the same thing while copy/pasting this code :)

Reply:
Probably needs to be broken up into multiple files as well, any code file above 1000+ lines is an immediate code smell

Reply:
We are about to drop half of all the code and tests in db as we will remove old schema support. But even after that, I agree - migrations could probably be extracted for example.